src/transport/raw/TCP.h

```diff
@@ -225,6 +225,9 @@ class DLL_EXPORT TCPBase : public Base
      */
     CHIP_ERROR ProcessSingleMessage(const PeerAddress & peerAddress, ActiveConnectionState * state, uint16_t messageSize);
 
+    // Release an active connection from the pool corresponding to the passed TCPEndPoint.
```

Comment:
Is the connection corresponding to the endpoint, or the pool? This says the pool is...

Reply:
Ah. I see how it reads. The connection corresponds to the endpoint. Will reword